helpers/hversion.py

```diff
@@ -217,6 +217,35 @@ def _check_version(code_version: str, container_version: str) -> bool:
     return is_ok
 
 
+def bump_version(version: str, bump_type: str = "minor") -> str:
```

Comment:
need a * or let's make it mandatory. Default params just create uncertainty

Reply:
Added *.